03_launch_mgmt_cluster.sh

```diff
@@ -29,13 +29,22 @@ M3PATH="${M3PATH:-${GOPATH}/src/github.com/metal3-io}"
 BMOPATH="${BMOPATH:-${M3PATH}/baremetal-operator}"
 RUN_LOCAL_IRONIC_SCRIPT="${BMOPATH}/tools/run_local_ironic.sh"
 CAPM3PATH="${CAPM3PATH:-${M3PATH}/cluster-api-provider-metal3}"
+CAPM3REPO="${CAPM3REPO:-https://github.com/metal3-io/cluster-api-provider-metal3.git}"
+CAPM3RELEASEPATH="${CAPM3RELEASEPATH:-https://github.com/metal3-io/cluster-api-provider-metal3/releases/latest}"
+CAPM3_RELEASE=$(curl -ksL "${CAPM3RELEASEPATH}" \
+                   | grep -m 1 "releases/download" \
+                   | cut -d \" -f 2 | tr -d \" \
```

Comment:
if you use jq, you can reduce this to a single command after a pipe

Reply:
Done